arm-sql/2015-05-01-preview/swagger/encryptionProtectors.json

```diff
@@ -0,0 +1,355 @@
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "List encryption protectors by server": {
+            "$ref": "../examples/EncryptionProtectorList.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/encryptionProtector/{encryptionProtectorName}": {
+      "get": {
+        "tags": [
+          "EncryptionProtectors"
+        ],
+        "description": "Returns the server encryption protector.",
+        "operationId": "Servers_GetEncryptionProtector",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "encryptionProtectorName",
+            "in": "path",
+            "description": "The name of the encryption protector to be retrieved.",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "current"
+            ],
+            "x-ms-enum": {
+              "name": "EncryptionProtectorName",
+              "modelAsString": true
+            }
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the specified server encryption protector.",
+            "schema": {
+              "$ref": "#/definitions/EncryptionProtector"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 InvalidKeyName - An invalid value was given for the server key name.\n\n * 400 InvalidKeyType - The create server key type is not supported.\n\n * 400 InvalidKeyUpsertRequest - The create server key request does not exist or has no properties object.\n\n * 400 InvalidEncryptionProtectorName - The encryption protector key name is not supported.\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 ResourceNotFound - The requested resource was not found."
+          }
+        },
+        "x-ms-examples": {
+          "Get the encryption protector": {
+            "$ref": "../examples/EncryptionProtectorGet.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "EncryptionProtectors"
+        ],
+        "description": "Updates an existing encryption protector.",
+        "operationId": "Servers_CreateOrUpdateEncryptionProtector",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
```

Comment:
Why is this? You previously told us that SubscriptionId and ApiVersion should be together since they are client params, not method params. Did this change?